File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1847.py
l = [int(i) for i in input().split()]

a = l[0]
b = l[1]
c = l[2]

feliz = False

if(a>b and b<=c): feliz=True;
elif(b>a and c>b and c-b>=b-a): feliz=True;
elif(a>b and b>c and b-c<a-b): feliz=True;
elif(a==b and c>b): feliz=True;

if(feliz): print(":)");
else: print(":(")

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1847.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');
    
var a = parseInt(lines.shift());
var b = parseInt(lines.shift());
var c = parseInt(lines.shift());

feliz=false
    
if(a>b && b<=c) feliz=true
else if(b>a && c>b && c-b>=b-a) feliz=true
else if(a>b && b>c && b-c<a-b) feliz=true
else if(a==b && c>b) feliz=true
    
if(feliz) console.log(":)");
else console.log(":(");
